docs/articles/faq/README.md

```diff
@@ -32,30 +33,25 @@ This proxy injects the driver script that emulates user actions into the tested
 You can read about this in our [forum](https://testcafe-discuss.devexpress.com/t/why-not-use-selenium/47).
 Feel free to ask for more details.
 
-### What is the difference between [TestCafe Studio](https://www.devexpress.com/products/testcafestudio/) and [open-source TestCafe](https://devexpress.github.io/testcafe)? Has [TestCafe v2015.1](https://testcafe.devexpress.com) been deprecated?
-
-Both TestCafe Studio and open-source TestCafe share the same core features:
-
-* No need for WebDriver, browser plugins or other tools.
-* Cross-platform and cross-browser out of the box.
-
-[TestCafe](https://devexpress.github.io/testcafe)  
-*first released in 2016, free and open-source node.js application*
-
-* You can write tests in the latest JavaScript or TypeScript.
-* Clear and flexible [API](https://devexpress.github.io/testcafe/documentation/test-api/) supports ES6 and [PageModel pattern](https://devexpress.github.io/testcafe/documentation/recipes/using-page-model.html).
-* Stable tests due to the [Smart Assertion Query Mechanism](https://devexpress.github.io/testcafe/documentation/test-api/assertions/#smart-assertion-query-mechanism).
-* Tests run fast due to intelligent [Automatic Waiting Mechanism](https://devexpress.github.io/testcafe/documentation/test-api/waiting-for-page-elements-to-appear.html) and [Concurrent Test Execution](https://devexpress.github.io/testcafe/documentation/using-testcafe/common-concepts/concurrent-test-execution.html).
-* Easy integration: it is a node.js solution with CLI and reporters for popular CI systems.
-* You can extend it with [plugins](https://github.com/DevExpress/testcafe#plugins) and other Node.js modules.
-
-[TestCafe Studio](https://www.devexpress.com/products/testcafestudio/)  
-*Preview released in 2018, commercial desktop application*  
-
-* Based on the open-source TestCafe, and supports its major features.
-* You can record tests or edit them as JavaScript or TypeScript code.
-* New [Visual Test Recorder](https://docs.devexpress.com/TestCafeStudio/400165/guides/record-tests) and [IDE-like GUI](https://docs.devexpress.com/TestCafeStudio/400181/user-interface/code-editor) to record, edit, run and debug tests.
-* Currently available as a free preview version.
+### What is the difference between [TestCafe Studio](https://www.devexpress.com/products/testcafestudio/) and [open-source TestCafe](https://devexpress.github.io/testcafe)?
+
+| &nbsp; | [TestCafe](https://devexpress.github.io/testcafe) | [TestCafe Studio](https://www.devexpress.com/products/testcafestudio/)  |
+| ------ |:-------------------------------------------------:|:-----------------------------------------------------------------------:|
+| No need for WebDriver, browser plugins or other tools | &#10003; | &#10003; |
```

Comment:
Nice!
Only absent `Easy integration with popular CI systems` in Studio colon looks sad for me. Maybe call this statement just `Integration with popular CI systems` and check it for both Open source and Studio?)
We also can tick `*` for `Extendable with plugins and other Node.js modules` for Studio and give e.g. following description: `Studio supports custom reporter plugins[link to docs] and you can use any Node js modules in coded tests`

Reply:
I've just thought, maybe it would be useful for current/possible testcafe users to have powerful and flexible [Run Configuration Manager](https://docs.devexpress.com/TestCafeStudio/400189/user-interface/run-configurations-dialog)? Should we mention about it?